.github/workflows/ci.yaml

```diff
@@ -3,6 +3,19 @@ name: CI
 on: [push, pull_request]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
This job doesn't have a `matrix`, so `matrix.python-version` is empty. You appear to have got the default for `actions/setup-python@v2` but it's not very clear what its default is.

Probably better to hardcode 3.10 or something like that.

Reply:
You’re right; I copied this from another project which had a matrix. Hardcoded to Python 3.10 now.